### PR TITLE
Update for 2.1 (new UI + association to product model)

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/product-associations.md [hunk 1]
```diff
@@ -15,15 +15,25 @@ A product may be associated with none or several other products or product group
 This type of relationship enables you to define relationships like an alternative product or a substitution between products for example.
 :::
 
+:::info
+Since the 2.1, a product can be associated to a [product model](/articles/what-about-products-variants.html#what-is-a-product-model).
+:::
+
+:::warning
+Only products can have associations, [product models](/articles/what-about-products-variants.html#what-is-a-product-model) have no association.
```
Comment (KmilleF) [inline content/md/products-super-power/product-associations.md]: Can't be associated or can't have association?

File: content/md/products-super-power/product-associations.md [hunk 2]
```diff
@@ -32,12 +42,56 @@ The green icon in front of the name of the association type shows that the produ
 
 The associated products can be identified by the input `Yes` in green in the `Is associated` column of the grid. The boxes are also checked.
 
+## View the product’s associations (_since 2.1_)
+
+Only the associated products or [product models](/articles/what-about-products-variants.html#what-is-a-product-model) are displayed in the `Associations` tab.
+
+![View the associated products](../img/Products_ProductsAssociated21.png)
+
+By default, if the product has no association for the association type selected, nothing is displayed in the "Associations" tab.
```
Comment (KmilleF) [inline content/md/products-super-power/product-associations.md]: "for the selected association type"

File: content/md/products-super-power/product-associations.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/product-associations.md]: effectivement c'est ambigu

File: content/md/products-super-power/product-associations.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/product-associations.md]: rah j'inverse toujours

**Pull request merged**